validator/testcases/regex.py

```diff
@@ -537,15 +537,15 @@ class Gecko53RegexTests(CompatRegexTestHelper):
     def tests(self):
         yield self.get_test_bug(
             1321556,
-            r'\burlbarBindings.xml\#splitmenu\b',
+            r'\burlbarBindings\.xml#splitmenu\b',
             'The splitmenu element has been removed.',
             'The splitmenu element has been removed.',
             log_function=self.err.warning,
             compat_type='error')
 
         yield self.get_test_bug(
             1331296,
-            r'\b[^-]*-moz-calc\b',
+            r'-moz-calc\b',
```

Comment:
I think I did something stupid here, `-` can be used as a "from to" (e.g `a-zA-Z`) and apparently I didn't escape it properly. I wonder if that already helps, there was a reason I chose this regex since a few others (e.g `\b-moz-calc\b`) didn't properly match. Generally I'm 👍 for your approach but I think the actual problem can be fixed via `\b[^\-]*-moz-calc\b`